docs-site/docs/docs/about-menu.md

```diff
@@ -0,0 +1,43 @@
+The about menu provides a deeper view into your instance of Polynote. 
+
+![Settings Menu](images/settings-menu.png)
```

Comment:
Since this is the `About` pane, I suggest moving this image into the `About` section and adding some more screenshots for each section if that's not too much trouble.